### PR TITLE
Ignore ssl and report errors

### PR DESCRIPTION
**Pull request by iloveitaly**
@sthnaqvi Follow up to https://github.com/sthnaqvi/follow-redirect-url/pull/8

Test pass locally, will monitor to see if CI runs properly.

### CODE REVIEW

File: follow-redirect-url.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 "use strict";
 
 const fetch = require("node-fetch");
+const https = require("https");
+const http = require("http");
+fetch.Promise = Promise;
```
Comment (sthnaqvi) [inline follow-redirect-url.js:6]: `fetch.Promise = Promise;` is not required here fetch already has inbuilt promise.

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [14.x, 15.x, 16.x]
```
Comment (sthnaqvi) [inline .github/workflows/node.js.yml:19]: some packages in this project did not support 15.x and 16.x, That's why I removed those node versions from the check and also added engine supported versions in `package.json` file

File: .travis.yml [hunk 3]
```diff
@@ -1,4 +0,0 @@
-language: node_js
```
Comment (sthnaqvi) [inline .travis.yml:1]: Why removed this file ?

File: .github/workflows/node.js.yml (hunk 2 above)
Comment (iloveitaly) [reply .github/workflows/node.js.yml:19]: All of the tests are passing for me. I think we should keep this in.

File: follow-redirect-url.js (hunk 1 above)
Comment (iloveitaly) [reply follow-redirect-url.js]: Removed! Not sure how this made its way in.

**Pull request merged**